src/service/parallax-impl.js

```diff
@@ -187,9 +136,7 @@ export class ParallaxElement {
    */
   shouldUpdate(viewport) {
     const viewportRect = viewport.getRect();
-    const elementRect = viewport.getLayoutRect(this.element_);
-    elementRect.top -= viewportRect.top;
-    elementRect.bottom = elementRect.top + elementRect.height;
+    const elementRect = this.element_/*OK*/.getBoundingClientRect();
```

Comment:
EEK! This is a horrible performance trap! We should be using `#getLayoutBox` when possible.